app/components/ListItemSwitch/ListItemSwitch.styles.ts

```diff
@@ -0,0 +1,25 @@
+import { StyleSheet } from "react-native";
+
+import { Colors } from "style";
+
+export default StyleSheet.create({
+  container: {
+    flexDirection: "row",
+    justifyContent: "space-between",
+    alignItems: "center",
+    paddingVertical: 18,
+    flex: 1,
+  },
+  switch: {
```

Comment:
switch style can be deleted. Instead, remove `paddingVertical: 18,` from the container and create a text style like this : 
```
  text {
    paddingVertical: 18,
  },
```
And then : 
```
      <Text.Secondary style={styles.text} numberOfLines={1}>
        {title}
      </Text.Secondary>`
```
This should fix your alignments issues ;)

Reply:
Oh yeah that worked. Comparing ListItem and its styles to ListItemSwitch and its styles now to try and figure out why it worked. Could we make the same change to ListItem (move the padding to the container component's \<Text\> child) to achieve the same result? And moreover, why does moving the padding inward help center the switch? Is it because the container style's `alignItems: center`, and when you add padding to the container it changes the center slightly (that doesn't make a whole lot of sense, but I'm not sure what's going on)?

Reply:
When the components inside a parent component are almost the same size, then applying the padding to the parent is usually the best way to go. However, when you have different size component inside one parent which has the padding, then you can have some kind of "overflow" like in ListItemSwitch. In my experience, the best way to visualize/debug that is to put different background color to the different components, it helps understanding how to apply constraints in a "lego/tetrix" way. So yeap, I think it's thanks to the `alignItems: center` that child components are aligned now (I did a quick debugging yesterday so not sure 100%)

Reply:
Oh okay, that makes sense. I'm guessing then you want to apply the padding to the component with the smallest size in the target axis? I should probably review the box model. Also changing the colors totally helped visualize that and enabled me to actually play around with it meaningfully instead of pure guess-and-check. Thanks!